spec/site_prism/page_objects/page/result_filters/subject_page.rb

```diff
@@ -15,6 +15,7 @@ class SubjectAreaSection < SitePrism::Section
       class SubjectPage < SitePrism::Page
         set_url "/results/filter/subject{?query*}"
         sections :subject_areas, SubjectAreaSection, '[data-qa="subject_area"]'
+        section :send_area, SubjectAreaSection, '[data-qa="send_area"]'
```

Comment:
I question whether or not SEND is a subject area from a data modelling perspective, but this should be fine for now